src/components/Payroll/PayrollConfiguration/PayrollConfigurationPresentation.tsx

```diff
@@ -98,16 +100,15 @@ export const PayrollConfigurationPresentation = ({
 
       <Flex flexDirection="column" gap={16}>
         {/* TODO: Replace with actual deadline information from payroll data */}
-        <Alert label="Payroll Deadline" status="info">
-          To pay your employees with direct deposit on the check date, you&apos;ll need to run
-          payroll by the deadline.
+        <Alert label={t('alerts.payrollDeadline.label')} status="info">
+          {t('alerts.payrollDeadline.message')}
         </Alert>
 
         {/* TODO: Replace with actual skipped employees list from payroll data */}
-        <Alert label="Skipped Employees" status="warning">
+        <Alert label={t('alerts.skippedEmployees.label')} status="warning">
           <ul>
-            <li>Employee address not verified</li>
-            <li>Employee address not verified</li>
+            <li>{t('alerts.skippedEmployees.employeeAddressNotVerified')}</li>
+            <li>{t('alerts.skippedEmployees.employeeAddressNotVerified')}</li>
```

Comment:
This text may be dynamic, maybe worth holding off on adding it until we wire it up in the ticket itself? i'd also be ok just removing these components until they actually get implemented